per/serializers.py

```diff
@@ -1046,6 +1049,29 @@ class Meta:
         exclude = ("learning", "type", "organization", "sector", "per_component")
 
 
+class AppealDetailSerializer(serializers.ModelSerializer):
+    atype = serializers.SerializerMethodField()
```

Comment:
Maybe use this instead? SerializerMethodField doesn't provide proper schema
```suggestion
    atype_display = serializers.CharField(source='get_atype_display', read_only=True)
```

Reply:
This Serializer is not in use.